packages/video_player/video_player_web/test/video_player_web_test.dart

```diff
@@ -2,7 +2,6 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 @TestOn('browser')
```

Comment:
We've been migrating tests to run with `flutter drive` instead of `flutter test --platform`, but that can be done as a separate PR.